pages/linux/sv.md

```diff
@@ -0,0 +1,15 @@
+# sv
+
+> Control a running runsv service.
+
+- Start a service:
+
+`sv up {{path/to/service}}`
```

Comment:
Doesn't this require root ?

Reply:
Not necessarily, if the service runs as the user giving the command, but I'll add `sudo`.